samples/bootloader/src/main.c

```diff
@@ -26,6 +26,29 @@
 #endif
 #endif
 
+
+#ifdef CONFIG_SB_DISABLE_FLASH_PATCH
+/* Copied from nrf_nvmc.c */
```

Comment:
Do we need to have this here? IMO it does not matter where it comes from.

Reply:
Removed